src/floris/simulation/wake_deflection/jimenez.py

```diff
@@ -63,9 +61,9 @@ def function(
         yaw_i: np.ndarray,
         turbulence_intensity_i: np.ndarray,
         ct_i: np.ndarray,
+        rotor_diameter: np.ndarray,
```

Comment:
Appropriate to use `_i` here?

Reply:
Ah I see you did in Gauss. I'll make that change.

Reply:
Thanks for doing that!